test/functional/apps/dashboard_elements/controls/controls_callout.ts

```diff
@@ -57,6 +76,24 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await testSubjects.missingOrFail('controls-empty');
       });
 
+      it('deleting all controls shows the emoty control callout again', async () => {
+        await dashboardControls.deleteAllControls();
+        await testSubjects.existOrFail('controls-empty');
+      });
+
+      it('hide callout when hide announcement setting is true', async () => {
+        await dashboard.clickQuickSave();
+        await dashboard.gotoDashboardLandingPage();
+        await kibanaServer.uiSettings.update({ hideAnnouncements: true });
+        await browser.refresh();
```

Comment:
When trying to set the `hideAnnouncements` setting and refresh the browser **while on a dashboard**, I kept getting a "Reload site?" warning from Chrome **even when** I saved - this meant I couldn't get the dashboard to actually refresh so this test would fail. Navigating to the dashboard landing page to set the setting, refresh, then go back to the dashboard seems to do the trick 🤷

Reply:
That's so strange, I just tested this locally and didn't run into this, can reload without any warnings after the dashboard is saved....

Reply:
Right? It works fine if I recreate the steps manually, but for some reason, when running the functional test, hitting the `browser.refresh()` when on the dashboard would cause the warning.

Reply:
Ah well, this is a fine workaround imo.